File: front/src/app/service/parecer/parecer.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Parecer } from 'src/app/modelos/parecer';
import { Util } from 'src/app/util';
import { Response } from 'src/app/modelos/response';

@Injectable({
  providedIn: 'root'
})
export class ParecerService {

  constructor(public http: HttpClient) { }

  public cadastrar(parecer: Parecer) {
    return this.http.post<Response>(Util._url + 'api-parecer/parecer', parecer);
  }
  public listarParecerDoProcesso(idProcesso) {
    return this.http.get<Response>(Util._url + 'api-parecer/parecer?idProcesso=' + idProcesso);
  }

}

File: back/src/main/java/br/com/softplan/parecer/service/ParecerService.java
package br.com.softplan.parecer.service;

import java.util.List;

import br.com.softplan.parecer.modelos.Parecer;

public interface ParecerService {

	public Parecer incluirParecer(Parecer parecer);

	public List<Parecer> listarParecerPorProcesso(Integer idProcesso);
}
